src/makeRobotAccountant.js

```diff
@@ -6,7 +6,26 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 3;
+  let evenCount = 0;
+
+  const getSum = (num1) => {
+    const adding = (num2) => {
+      count--;
+
+      if (count < 0 && evenCount % 2 === 0) {
```

Comment:
You could implement this glitching functionality with only one variable - `count`)

Reply:
You're right)